### PR TITLE
Return sentinel errors to help handling by callers

### PR DESCRIPTION
**Pull request by jtheopillus**
Hi Eppo team,
my team wants to be able to handle the cases where flag is not found/not enabled better and I think it'd be more convenient to define these errors as (public) vars so they can be handled more nicely with `errors.Is( ... )`

thanks!

### CODE REVIEW

**Review by felipecsl — APPROVED** [review]

File: eppoclient/errors.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package eppoclient
+
+import "errors"
+
+var (
```
Comment (aarsilv) [inline eppoclient/errors.go:5]: 🙌

**Review by aarsilv — APPROVED** [review]
Great idea!

Looks good to me! 

Note: The failing tests are unrelated to this change. They are for a known contextual multi-armed bandit edge case (no provided actions) that we changed how we want to handle and need to update this SDK for.

**Comment by jtheopillus** [comment]
> Great idea!
> 
> Looks good to me!
> 
> Note: The failing tests are unrelated to this change. They are for a known contextual multi-armed bandit edge case (no provided actions) that we changed how we want to handle and need to update this SDK for.

great! then could you please merge it and release a new patch? @aarsilv @felipecsl 
thanks, appreciate it :)

**Review by rasendubi — APPROVED** [review]
Looks good to me. @jtheopillus thanks for your contribution!

I've added one commit to bump the SDK version
